pkgs/linux-cachyos/config.nix

```
@@ -0,0 +1,89 @@
+{ lib, ... }:
+with lib.kernel; {
+  LOCALVERSION = freeform "-cachyos-bore";
+  EXPERT = yes;
+  WERROR = no;
+
+  # Bore scheduler
+  SCHED_BORE = yes;
+
+  # Tick to 750hz
+  HZ = freeform "500";
+  HZ_500 = yes;
```

```suggestion
  HZ_500 = yes;
  HZ_1000 = no;
```

We have to remove the upstream value.